### PR TITLE
EVG-7974 Add support for selecting multiple tasks in the configure tab

### PR DESCRIPTION
**Pull request by khelif96**
image
You can select multiple  variants by command clicking on them

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -69,6 +69,10 @@ module.exports = {
     },
   },
   rules: {
+    "@typescript-eslint/no-unused-vars": [
+      errorIfStrict,
+      { vars: "all", args: "after-used", ignoreRestSiblings: true },
+    ],
```
Comment (tgrander) [inline .eslintrc.js:75]: what's dis?

File: cypress/integration/patch/configure-patch.ts [hunk 2]
```diff
@@ -178,6 +178,64 @@ describe("Configure Patch Page", () => {
             .should("eq", $tasks.length);
         });
     });
+    describe("Selecting multiple build variants", () => {
+      it("Selecting multiple build variants should work", () => {
+        cy.get('[data-cy-name="linux-docker"]').click();
```
Comment (tgrander) [inline cypress/integration/patch/configure-patch.ts:183]: why not use `.dataCy` here?

File: .eslintrc.js (hunk 1 above)
Comment (khelif96) [reply .eslintrc.js:75]: This was already here i just moved it. So the plugins are in alphabetical order

File: cypress/integration/patch/configure-patch.ts (hunk 2 above)
Comment (khelif96) [reply cypress/integration/patch/configure-patch.ts:183]: the tag was a `data-cy-name` not a `data-cy` so i cant use `.dataCy` for it.

**Pull request merged**